protontricks: refresh page (#8492)
tldr-pages/tldr: pages/linux/protontricks.md

# protontricks

> A simple wrapper that runs Winetricks commands for Proton enabled games.
> More information: <https://github.com/Matoking/protontricks>.

- Show the protontricks help message:

> `protontricks --help`

Reviewer: Put help code examples to the end.

- Run a command within a game's installation directory:

`protontricks -c {{command}} {{appid}}`

> - List all installed games:

Reviewer: ```suggestion
- [l]ist all installed games:
```

You can consider option case as more important thing than letter case in code description. That's why you can change `L` to `l` and not use `[L]`.

`protontricks -l`

> - Search for a game's App ID by name:

Reviewer: ```suggestion
- [s]earch for a game's App ID by name:
```

Please use mnemonics as much as possible. If you don't know which mnemonic to use we can help. ;)

Author: > Please use mnemonics as much as possible. If you don't know which mnemonic to use we can help. ;)

Sure, I will try to use more mnemonics in future, I was just confused about whether to use them in commands with single arguments.

> - Search for a game's App ID by entering it's name:

Reviewer 2: **fixup:** "it's" is used as a contraction of "it is", "its" is the possessive form. We can also avoid this entirely:

```suggestion
- Search for a game's App ID by name:
```

> `protontricks -s {{Game_Name}}`

Reviewer: ```suggestion
`protontricks -s {{game_name}}`
```

Just snake_case, nothing more is permitted unless it's a placeholder sample value.
